entity-service/src/test/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationRepositoryTest.kt

```diff
@@ -596,10 +597,10 @@ class Neo4jAuthorizationRepositoryTest : WithNeo4jContainer {
             listOf(firstEntity.id, secondEntity.id, thirdEntity.id, fourthEntity.id),
             result.second.map { it.id }
         )
-        assertTrue(result.second.find { it.id == fourthEntity.id }?.specificAccessPolicy == AUTH_READ)
-        assertTrue(result.second.filter { it.id != fourthEntity.id }.none { it.specificAccessPolicy != null })
-        assertTrue(result.second.find { it.id == secondEntity.id }?.right == R_CAN_ADMIN)
         assertTrue(result.second.all { it.type.size == 1 && it.type[0] == "Beekeeper" })
+        assertTrue(result.second.find { it.id == secondEntity.id }?.right == R_CAN_ADMIN)
```

Comment:
why did you move these 3 lines (it just creates noise, to be avoided without a good reason to do it)?

Reply:
it's a mistake sorry